apiclient/harvester_api/managers/images.py

```diff
@@ -11,7 +11,8 @@ class ImageManager(BaseManager):
     DOWNLOAD_fmt = "v1/harvester/harvesterhci.io.virtualmachineimages/{ns}/{uid}/download"
     _KIND = "VirtualMachineImage"
 
-    def create_data(self, name, url, desc, stype, namespace, display_name=None, storageclass=None):
+    def create_data(self, name, url, desc, stype, namespace, imageChecksum,
```

Comment:
Should have a default value or it will break existing `images.create_data` calls cuz they do not specify checksum.
e.g.
https://github.com/harvester/tests/blob/ba771f73e3c3e504449707b5f8d625a70946ffd9/harvester_e2e_tests/apis/test_images.py#L105

Reply:
Changed to default of None.